src/file-5.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:27960/organizations/default-organization/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:27960/project/issues?id=sample&branch=feature6&issues=AWFGJK0X2p3WLj3Iyd2P&open=AWFGJK0X2p3WLj3Iyd2P)

[ISSUE_KEY]: AWFGJK0X2p3WLj3Iyd2P